backend/schemas/scripts/render.py

```diff
@@ -28,7 +28,8 @@
 
 MAGIC_MAX_ROWS = 1048576
```

Comment:
I have a suspicion we should get rid of the `MAGIC_MAX_ROWS` and `MAX_ROWS` variables? `XLSX_MAX_ROWS` is repetitive w/ `MAGIC`, but it is named better.

Reply:
Yep. I... suspect the 3000 comes from me discovering that slowdown at some point in the past. Thank you for the reminder. (Also, all these questions/comments are largely conversation... not critique.)